cmd/ocm-backplane/managedJob/createManagedJob.go

```diff
@@ -82,6 +90,26 @@ func runCreateManagedJob(cmd *cobra.Command, args []string) (err error) {
 		return err
 	}
 
+	if options.manager {
+		mcid, clusterName, _, err := ocm.DefaultOCMInterface.GetManagingCluster(bpCluster.ClusterID)
```

Comment:
```suggestion
		if mcid, clusterName, _, err := ocm.DefaultOCMInterface.GetManagingCluster(bpCluster.ClusterID); err == nil {
```

The first case can be handled in a single line.